src/deprecation.h

```diff
@@ -10,8 +10,8 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 1661960;
-static const int RELEASE_TO_DEPRECATION_WEEKS = 2;
+static const int APPROX_RELEASE_HEIGHT = 1663550;
```

Comment:
The mainnet block height as of this comment was 1663437, so 113 blocks before this height (which is about right for at least a few hours of CI and release work).